models/migrations/v104.go

```diff
@@ -1,7 +1,6 @@
 // Copyright 2019 The Gitea Authors. All rights reserved.
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
-
```

Comment:
why remove space between licence and package

Reply:
migration v104 & 106 should not be touched if not related ...

Reply:
Will fix